google-api-client-assembly/dependencies/APACHE-LICENSE.txt

```diff
@@ -1,202 +0,0 @@
-

```

Comment:
We probably still need to keep the license files in here. These are the collected license files of the dependencies used and are required if we redistribute.